From 8c3d531e9d8e4e4930fe34c3d1eaa9325f914762 Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Tue, 16 Sep 2014 15:30:39 +0200
Subject: [PATCH] Downstream patch to the list of unremovable system apps

Replace Epiphany with Firefox, and drop gnome-dictionary and
gnome-system-log that aren't included in the default Fedora Workstation
install.
---
 data/modulesets/gnome.xml | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/data/modulesets/gnome.xml b/data/modulesets/gnome.xml
index 7fe8cce..e3e1ed7 100644
--- a/data/modulesets/gnome.xml
+++ b/data/modulesets/gnome.xml
@@ -23,12 +23,10 @@
 <moduleset name="system">
   <module type="application">empathy.desktop</module>
   <module type="application">eog.desktop</module>
-  <module type="application">epiphany.desktop</module>
   <module type="application">evince.desktop</module>
+  <module type="application">firefox.desktop</module>
   <module type="application">gnome-calculator.desktop</module>
-  <module type="application">gnome-dictionary.desktop</module>
   <module type="application">gnome-disks.desktop</module>
-  <module type="application">gnome-system-log.desktop</module>
   <module type="application">gnome-system-monitor.desktop</module>
   <module type="application">gnome-terminal.desktop</module>
   <module type="application">gucharmap.desktop</module>
-- 
2.1.0

